docs/source/noise/column_noise.rst

```diff
@@ -302,4 +302,45 @@ This noise type is called :code:`make_typos` in the configuration. It takes two
     - The probability of each character (which we call a "token") having a typo
       **given that the cell is being considered for this noise type**.
       One way to think about this is the probability of a typo on any given character when the value is being typed carelessly.
+    - 0.1 (10%)
+
+Make Optical Character Recognition (OCR) errors
+--------------------------------------------------
+
+Optical character recognition (OCR) is when a string is misread for another string that is visually similar. Some common examples are
+‘S’ instead of ‘5’ and ‘m’ instead of ‘iii’.
+
+pseudopeople defines the possible OCR substitutions using `this CSV file <https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/ocr-variations-upper-lower.csv>`_, which was produced by the `GeCO project <https://dl.acm.org/doi/10.1145/2505515.2508207>`_. In the file, the first column is the real string (which we call a "token") and the second column is what it could be misread as (a "corruption").
+The same token can be associated with multiple corruptions.
+
+To implement this, we first select the rows to noise, as in other noise functions.
```

Comment:
```suggestion
To implement this, we first select the rows to noise, as in other noise types.
```

Reply:
Note: This paragraph is a bit more technical than what we have for other noise types. It would be nice to streamline by:

- Removing discussion of the parameters, just explaining the overall process, and letting the parameters table describe which aspects are configurable.
- Pulling the part about token order and overlapping tokens out into a separate note (after the table?) since it is not critical to understand.

But I don't know if we need to do that in this PR.